include/libdnf5/base/goal_elements.hpp

```diff
@@ -82,7 +107,10 @@ enum class GoalProblem : uint32_t {
     SOLVER_PROBLEM_STRICT_RESOLVEMENT = (1 << 13),
     WRITE_DEBUG = (1 << 14),
     UNSUPPORTED_ACTION = (1 << 15),
-    MULTIPLE_STREAMS = (1 << 16)
+    MULTIPLE_STREAMS = (1 << 16),
+    MODULE_SOLVER_ERROR = (1 << 17),
```

Comment:
This is not a problem but an idea, what about to add a doc strings for each item?